api/client/src/main/java/org/projectnessie/client/http/impl/jdk11/JavaResponseContext.java

```diff
@@ -75,6 +84,21 @@ public URI getRequestedUri() {
     return response.uri();
   }
 
+  public void close(Exception error) {
+    if (error != null) {
+      try {
+        LOGGER.debug(
+            "Closing unprocessed input stream for {} request to {} delegating to {} ...",
+            response.request().method(),
+            response.uri(),
+            response.body());
+        response.body().close();
+      } catch (IOException e) {
+        // ignore
```

Comment:
why not log it?

Reply:
I did not change this part of the code, only moved it.